src/rest/impl/contracts.service.ts

```diff
@@ -78,18 +79,30 @@ export default class ContractsServiceImpl extends ContractsService {
       return;
     }
 
-    const deliveryPlace = await models.findDeliveryPlaceByExternalId(contract.deliveryPlaceId);
-    const proposedDeliveryPlace = await models.findDeliveryPlaceByExternalId(contract.proposedDeliveryPlaceId || contract.deliveryPlaceId);
-    const itemGroup: ItemGroupModel = await models.findItemGroupByExternalId(contract.itemGroupId);
+    const deliveryPlace: DeliveryPlaceModel = await models.findDeliveryPlaceByExternalId(contract.deliveryPlaceId);
+    if (!deliveryPlace) {
+      this.sendBadRequest(res, "Invalid deliveryPlaceId");
```

Comment:
Change this syntax to all invalid responses
```suggestion
      this.sendBadRequest(res, `Delivery place with ID ${deliveryPlaceId} could not be found!`);
```

Reply:
Done